proto/plan.proto

```diff
@@ -249,6 +249,7 @@ message ColDef {
 	bool primary        = 5;
 	int32 pkidx 		= 6;
 	string alias		= 7;
+	bool	isPrune		= 8;
```

Comment:
pls rename to :  is_prune
proto will transform to right name;

Reply:
ok, thank you for your advice